weave/trace/urls.py

```diff
@@ -33,3 +33,10 @@ def leaderboard_path(entity_name: str, project_name: str, object_name: str) -> s
 
 def redirect_call(entity_name: str, project_name: str, call_id: str) -> str:
     return f"{remote_project_root_url(entity_name, project_name)}/r/call/{call_id}"
+
+
+def evaluation_call_url(entity_name: str, project_name: str, call_id: str) -> str:
+    """Generate a URL for an evaluation call that links to the evaluations page."""
+    project_id = f"{entity_name}/{project_name}"
+    peek_path = f"/{project_id}/calls/{call_id}?hideTraceTree%3D1"
+    return f"{project_weave_root_url(entity_name, project_name)}/evaluations?view=evaluations_default&peekPath={peek_path}"
```

Comment:
we should not do this - encoding a peek path inside the dumped URL is quite ugly. The `/r/call` path above was invented exactly for this... we should just change what the "(r)edirect" is

Reply:
Can you show me what that would look like here?  I'm not sure I fully understand

Reply:
In core, `frontends/app/src/pages/RedirectPage/RedirectCall.tsx` controls where the `/r/call/id` gets routed. let's change that in the frontend so it does not create a link dependency between client and ui